README.md

```diff
@@ -102,15 +102,15 @@ spec:
 ```
 
 
-### Flow Code
+### Template Code
 
-The Camel route that defines the behavior of the Kamelet **MUST** be provided in YAML flow syntax.
+The Camel route that defines the behavior of the Kamelet **MUST** be provided in YAML flow syntax (in `.spec.template` parameter).
```

Comment:
When I google "YAML flow syntax" all the hits are about embedding json in yaml (i.e. curly brackets rather than  indentation).  None of the kamelets use format in the flow/tempate section, so I think you need to provide a link to a description of what you mean.